examples/combobox/combobox-autocomplete-none.html

```diff
@@ -53,7 +53,7 @@ <h2 id="ex_label">Example</h2>
                    aria-autocomplete="none"
                    aria-expanded="false"
                    aria-controls="cb1-listbox">
-            <button type="button" id="cb1-button" tabindex="-1" aria-label="Open">
+            <button type="button" id="cb1-button" tabindex="-1"  aria-label="State" aria-expanded="false" aria-controls="cb1-listbox" >
```

Comment:
This one controls the "Previous Searches" list, so the aria-label should reflect that.

```suggestion
            <button type="button" id="cb1-button" tabindex="-1"  aria-label="Previous Searches" aria-expanded="false" aria-controls="cb1-listbox" >
```

Reply:
Fixed